src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -2272,12 +2272,23 @@ public bool TryDequeueWaiter([MaybeNullWhen(false)] out TaskCompletionSourceWith
                 return false;
             }
 
-            public bool TryPeekRequest([MaybeNullWhen(false)] out HttpRequestMessage request)
+            public bool TryPeekUncancelledRequest(HttpConnectionPool pool, [MaybeNullWhen(false)] out HttpRequestMessage request)
```

Comment:
I'm not 100% sure but it seems that it might be
```suggestion
            public bool TryPeekUncanceledRequest(HttpConnectionPool pool, [MaybeNullWhen(false)] out HttpRequestMessage request)
```
because see e.g. Task.IsCanceled

Reply:
We're consistently inconsistent about this one 😆

`OperationCanceled`, `CancellationToken` ...

Happy to change it either way

Reply:
Cancel
Canceled
Canceling
Canceler
Cancellation

i.e. Cancellation is the only one with two l's.  There are a few places around the core libraries, from before we standardized on these spellings, that use different variations.  Ignore them :)